client/sites-dashboard/components/sites-table.tsx

```diff
@@ -48,6 +50,13 @@ export function SitesTable( { className, sites }: SitesTableProps ) {
 				</Row>
 			</thead>
 			<tbody>
+				{ isLoading && (
+					<>
+						<SitesTableRowLoading />
+						<SitesTableRowLoading />
+						<SitesTableRowLoading />
```

Comment:
An alternative would be to have each component know how to render an `isLoading` version of itself i.e.

```suggestion
						<SitesTableRow isLoading={ true } />
						<SitesTableRow isLoading={ true } />
						<SitesTableRow isLoading={ true } />
```

The row would render a `isLoading` version of itself and then pass the prop down to its sub-components. That way if we add another column we don't have to update both `<SitesTableRow>` _and_ `<SitesTableRowLoading>`. And as @vykes-mac pointed out, the thumbnail component will be responsible for knowing how big to render the placeholder version of itself. As it is we're already having to duplicate the `mobileHidden` code.

The downside is that `site` would become an optional prop and so there'd have to be a bunch of `undefined` handling inside `<SitesTableRow>` which is currently nice a clean from that sort of thing.

Reply:
I've decided to keep the `SitesTableRowLoading` separate from `SitesTableRow`, but now we don't need to update `SitesTableRowLoading` to add/remove the number of columns. We just need to change the prop `columns={5}` https://github.com/Automattic/wp-calypso/blob/172c2ed745b7ad9c4a986154494c0104c8956fdb/client/sites-dashboard/components/sites-table.tsx#L56-L64